crates/motoko/src/lib/vm_types.rs

```diff
@@ -514,12 +556,6 @@ pub struct NYI {
     pub line: u32,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub enum Signal {
-    Done(Value_),
-    Interruption(Interruption),
-}
-
```

Comment:
`Signal` was going to grow to capture errors from I/O.

Except now with the Motoko Dev Server, I am now expecting network I/O to happen outside of the VM, and for this traffic to selectively affect the VM from its public API for `Core`, which will grow to accommodate the Dev Server.